estate/models/real_estate.py

```diff
@@ -45,3 +61,57 @@ class RealEstate(models.Model):
         string='Tags'
     )
 
+    total_area = fields.Integer(compute="_compute_total_area")
+    best_offer = fields.Float(compute="_compute_best_offer")
+
+    @api.depends("offer_ids.price")
+    def _compute_best_offer(self):
+        for property in self:
+            property.best_offer = max(property.offer_ids.mapped('price')) if property.offer_ids else 0
```

Comment:
The built in `max()` supports default values already
```suggestion
            property.best_offer = max(property.offer_ids.mapped('price'), default=0)
```